@xen-orchestra/web/src/components/vm/VmGeneralInfo.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <UiPanel class="vm-general-info">
```

Comment:
```suggestion
  <UiCard class="vm-general-info">
```

`UiPanel` is not necessary there 
Prefer to use `UiCard` instead